### PR TITLE
Fix test output

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we clean up the test output generated when building `:server`.

Most of the pruned output represents some kind of an expected behavior, e.g. negative test cases.
Though some of it was an actual sign of a problem. Those problems are now fixed (with an exception of #853).

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, @alexander-yevsyukov, PTAL.

File: server/src/test/java/io/spine/server/CommandServiceTest.java [hunk 1]
```diff
@@ -33,6 +33,7 @@
 import io.spine.test.commandservice.CmdServDontHandle;
 import io.spine.testing.client.TestActorRequestFactory;
 import io.spine.testing.server.model.ModelTests;
+import io.spine.testlogging.MuteLogging;
```
Comment (alexander-yevsyukov) [inline server/src/test/java/io/spine/server/CommandServiceTest.java]: Please make this a part of Bast Testlib. This looks like a reusable utility.

**Comment by dmdashenkov** [comment]
@alexander-yevsyukov, PTAL again.

File: server/src/main/java/io/spine/server/command/model/CommandHandlerMethod.java [hunk 2]
```diff
@@ -68,33 +67,32 @@ protected Result toResult(CommandHandler target, Object rawMethodOutput) {
...
+            ensureNotEmptyIfNotProcessManager(filtered, producer);
             setMessages(filtered);
         }
 
         /**
          * Ensures that a command handler method produces one or more event messages.
          *
          * <p>The only allowed exception to this are {@link ProcessManager} instances returning
-         * {@link io.spine.server.model.Nothing Nothing} from command handlers.
+         * empty  from command handlers.
```
Comment (alexander-yevsyukov) [inline server/src/main/java/io/spine/server/command/model/CommandHandlerMethod.java]: Please remove extra space char.

**Pull request merged**